c2cwsgiutils/scripts/check_es.py

```diff
@@ -66,10 +68,19 @@ def _check_roundtrip() -> None:
     query = {"query": {"match_phrase": {"log.logger": logger_name}}}
     start = time.monotonic()
     while time.monotonic() < start + LOG_TIMEOUT:
-        for _ in range(10):
-            r = requests.post(SEARCH_URL, json=query, headers=SEARCH_HEADERS)
+        exception = None
+        for _ in range(int(os.environ.get("C2CWSGIUTILS_CHECKES_TRYNUMBER", 10))):
```

Comment:
One check, multiple checks. not checkes. So:
=> 
```suggestion
        for _ in range(int(os.environ.get("C2CWSGIUTILS_CHECKS_TRYNUMBER", 10))):
```

or C2CWSGIUTILS_CHECK_ES_TRYNUMBER ?